internal/uniqid/legacy.go

```diff
@@ -0,0 +1,51 @@
+package uniqid
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/ActiveState/cli/internal/errs"
+)
+
+const legacyPersistDir = "activestate/persist"
+
+func moveUniqidFile(destination string) error {
+	legacyDir, err := legacyStorageDir()
+	if err != nil {
+		return errs.Wrap(err, "Could not get legacy storage directory")
+	}
+
+	legacyUniqIDFile := filepath.Join(legacyDir, fileName)
+
+	// If the uniqID file does not not exist there is nothing to move
+	if !fileExists(legacyUniqIDFile) {
+		return nil
+	}
+
+	err = mkdir(filepath.Dir(destination))
+	if err != nil {
+		return errs.Wrap(err, "Could not create new persist directory")
+	}
+
+	err = os.Rename(legacyUniqIDFile, destination)
```

Comment:
Might be safer to do a copy, in case we don't have write permission to the file for some reason.